site/gdocs/components/Donors.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react"
+import { groupBy } from "@ourworldindata/utils"
+import { useDonors } from "../utils.js"
+
+export default function Donors({ className }: { className?: string }) {
+    const donors = useDonors()
+    if (!donors) return null
+
+    const donorsByLetter = groupBy(donors, (donor) => donor[0])
+    return (
+        <div className={className}>
+            <div className="col-start-2 span-cols-12">
+                <p className="donors-note">(Listed in alphabetical order)</p>
+                {Object.entries(donorsByLetter).map(([letter, donors]) => (
+                    <div key={letter}>
+                        <h3 className="donors-letter">{letter}</h3>
+                        <ul className="donor-list grid grid-cols-6 grid-lg-cols-5 grid-md-cols-4 grid-sm-cols-2">
```

Comment:
It might be better to use `columns` on the `ul` here so that multi-line names don't add a vertical gap to other columns.

https://github.com/user-attachments/assets/31c342da-d0df-45bf-9579-c6df167299f1

though you'll also wanna tweak the line-height and margins so that it's clear when a name does wrap, that it's all still one name.

Reply:
Thanks, I prefer the current approach or adding back list bullets. I'll ask Marwa about how she want to handle it.

Reply:
Marwa agrees with your proposal, changed.